src/PokemonTrainer.java

```diff
@@ -0,0 +1,26 @@
+import java.util.List;
+
+public class PokemonTrainer {
+    String  name;
+    List<Pokemon> pokemons;
+
+    PokemonTrainer(String name, List<Pokemon> pokemons) {
+        this.name = name;
+
+        this.pokemons = pokemons;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public List<Pokemon> getPokemons() {
+        return pokemons;
+    }
+
+    private void setName(String name) {
```

Comment:
Waarom is deze methode private? Het wordt niet binnen de PokemonTrainer klasse gebruikt en kan nu niet buiten deze klasse gebruikt worden. Effectief is deze methode nu dus onbruikbaar.